docs/source/release_notes/lts/52_0.rst

```diff
@@ -163,3 +163,176 @@ When compared to the last LTS, we had:
+  ``avocado_framework_plugin_varianter_yaml_to_mux``. Without it Avocado
+  does not require PyYAML, but you need it to support the parsing of
+  YAML files to variants (unless you use a different plugin with similar
+  functionality, which is now also possible).
+* The HTML result plugin is now also an optional plugin so one has to
+  install it separately.
+* The remote execution features (``--remote-hostname``, ``--vm-domain``,
+  ``--docker``) were also turned into optional plugins so if you need
+  those you need to install them separately.
+* Support for virtual environment (``venv``) was greatly improved and we
+  do encourage people who want to use ``pip`` to do that via this method.
+
+As for the supported ways:
+
+* Fedora/RHEL can use our custom repositories, either LTS-only or all
+  releases. Note that latest versions (non-lts) are also available
+  directly in Fedora and also in EPEL.
+* OpenSUSE - Ships the 36 LTS versions, hopefully they'll start
+  shipping the 52 ones as well (but we are not in charge of that
+  process)
+* Debian - The ``contrib/packages/debian`` script is still available,
+  although un-maintained for a long time
+* PyPI/pip - Avocado as well as all optional plugins are available in
+  PyPI and can be installed via ``pip install avocado-framework\*``,
+  or selectively one by one.
+* From source - Makefile target ``install`` is still available but it
+  does not install the optional plugins. You have to install them one
+  by one by going to their directory (eg. ``cd optional_plugins/html``
+  and running ``sudo python setup.py install``.
+
+As before you can find the details in :ref:`Installing Avocado`.
+
+Usage
+^^^^^
+
+.. note:: As mentioned in previous section some previously core
+   features were turned into optional plugins. Do check your install
+   script if some command described here are missing on your system.
+
+Most workflows should work the same, although there are few little
+changes and a few obsoleted constructs which are still valid, but you
+should start using the new ones.
+
+The hard changes which does not provide backward compatibility:
+
+* Human result was tweaked a bit:
+    * The ``TESTS`` entry (displaying number of tests) was removed as
+      one can easily get this information from ``RESULTS``.
+    * Instead of tests time (sum of test times) you get job time
+      (duration of the job execution) in the human result
+* Json results also contain some changes:
+    * They are pretty-printed
+    * As ``cancel`` status was introduced, json result contain an entry
+      of number of canceled tests (``cancel``)
+    * url was renamed to id (url entry is to be removed in ``53.0`` so
+      this is actually a soft change with a backward compatibility support)
+* The ``avocado multilex|variants`` does not expect multiplex YAML
+  files as positional arguments, one has to use ``-m|--mux-yaml``
+  followed by one or more paths.
```

Comment:
nop, on the other hand that's why you need the `--` to separate files and positional arguments ;-) (eg. `avocado run -m foo passtest.py` fails, you have to use `avocado run -m foo -- passtest.py`).